build_tools/cmake/iree_run_module_test.cmake

```diff
@@ -110,11 +110,11 @@ function(iree_run_module_test)
     elseif(_OUTPUT_FILE_TYPE STREQUAL ".txt")
       file(REAL_PATH "${_RULE_EXPECTED_OUTPUT}" _OUTPUT_FILE_SRC)
       # Process the text input to remove the line breaks.
-      file(STRINGS "${_OUTPUT_FILE_SRC}" _EXPECTED_OUTPUT ENCODING UTF-8)
-      string(REPLACE ";" " " _EXPECTED_OUTPUT_STR "${_EXPECTED_OUTPUT}")
+      file(READ "${_OUTPUT_FILE_SRC}" _EXPECTED_OUTPUT)
+      string(REPLACE "\n" " " _EXPECTED_OUTPUT_STR "${_EXPECTED_OUTPUT}")
       set(_EXPECTED_OUTPUT_STR "--expected_output=\"${_EXPECTED_OUTPUT_STR}\"")
       list(APPEND _RULE_RUNNER_ARGS ${_EXPECTED_OUTPUT_STR})
-      list(APPEND _SRC ${_OUTPUT_FILE_SRC})
+      list(APPEND _RUNNER_DATA ${_OUTPUT_FILE_SRC})
```

Comment:
Is it okay to use the real path here? (derived from line 111)

Reply:
The text content is added to the `_RULE_RUNNER_ARGS` so it is portable. Appending the real path of the text file in `_RUNNER_DATA` will make sure `run_android.x` copied the right file. All the file paths at `_RUNNER_DATA` is in absolute path for the same purpose.

Reply:
Oh, I'm confused with `_OUTPUT_FILE_SRC_PATH` and `_OUTPUT_FILE_SRC`. They look similar to me. Would be good if we can distinguish the names more (same to `_SRC` and `_RULE_MODULE_SRC`)

Reply:
Understood. I will add a commit to cleanup the names (absolute vs. relative)